doc/all.mk

```diff
@@ -106,10 +106,24 @@ doc/raddb/%.adoc: raddb/%
 	${Q}perl -pi -e 's/^# ([^ \t])/#  $$1/;s/^([ \t]+)# ([^ \t])/$$1#  $$2/;s/[ \t]+$$//' $^
 	${Q}./scripts/asciidoc/conf2adoc -a ${top_srcdir}/asciidoc -o $@ < $^
 
+#
+#	Converting *.adoc to *.html
+#
 doc/%.html: doc/%.adoc
 	@echo HTML $^
 	${Q}$(ASCIIDOCTOR) $< -a "toc=left" -b html5 -o $@ $<
-	${Q}perl -p -i -e 's/\.adoc/\.html/g' $@
+	${Q}perl -p -i -e 's/\.adoc/\.html/g; s/\/.html"/\/"/g' $@
```

Comment:
@alandekok That extra regex `s/\/.html"/\/"/g` is necessary because when we have hiperlinks like:

`[How-To](howto/) - How-To guides for performing common tasks.`

The `asciidoctor` convert appending a `\.html` to the end of link becoming `<a href="howto/.html">$text</a>`. so, the idea is basically to cleanup that `/\/.html`.

Reply:
That makes sense.  To make it a bit clearer, you can do:

```
${Q}perl -p -i -e 's,\.adoc,\.html,g; s,/.html",/",g' $@
```

which separates `/` as regex delimiter from `/` as filename delimiter